os/include/tinyara/pm/pm.h

```diff
@@ -199,6 +199,17 @@ enum pm_state_e {
 	PM_COUNT,
 };
 
+/* This enumeration provides all power management related wakeup source code. */
+
+typedef enum {
+	PM_WAKEUP_BLE,           /* BLE Interrupts */
+	PM_WAKEUP_WIFI,          /* Wifi Interrupts */
+	PM_WAKEUP_LOG_UART,      /* LOG UART Interrupts */
+	PM_WAKEUP_UART,          /* UART Interrupts */
+	PM_WAKEUP_GPIO,          /* Audio Interrupts */
+	PM_WAKEUP_HW_TIMER,      /* Timer Expiration */
+	PM_WAKEUP_UNKNOWN
```

Comment:
In case the wakeup source continue to grow in the future, can you please set the wakeup unknown to 0xFF or something else?